src/styles/calculator.css

```diff
@@ -0,0 +1,67 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #f5f5f5;
+  height: auto;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.button {
+  background-color: #e0e0e0;
+  height: 3rem;
+  padding: 0.8rem;
+  font-size: 1.5rem;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  border: none;
+  outline: none;
+}
+
+.button-container {
+  display: grid;
+  height: 14.5em;
+  gap: 0.5rem;
+  grid-template-rows: repeat(5, 1fr);
+}
+
+.row {
+  display: grid;
+  flex-direction: column;
+  gap: 0.5rem;
```

Comment:
- [ ] In order to match the given design let us kindly improve our styling; change `gap: 0.5rem` to `gap: 0.1rem`
After making these changes, here is how our app will look like compared to the original design
![Screenshot from 2023-08-25 23-43-05](https://github.com/mulugithub/math-magicians/assets/46388113/0ed406ba-cd43-4729-a6fa-029b1e90f5d5)

![Screenshot from 2023-08-25 23-38-54](https://github.com/mulugithub/math-magicians/assets/46388113/cb25226c-3994-4fa5-9980-38d746bdba6f)

Instead of what we have currently
![Screenshot from 2023-08-25 23-46-28](https://github.com/mulugithub/math-magicians/assets/46388113/39410528-7e38-4334-9828-d3d1d042c8ee)